.travis.yml

```diff
@@ -47,6 +47,7 @@ script:
     cmake -DCMAKE_PREFIX_PATH=$HOME/kokkos
           -DCabana_ENABLE_Serial=OFF ${CMAKE_OPTS[@]}
           -DCabana_ENABLE_MPI=ON
+          -DMPIEXEC_EXECUTABLE=/usr/bin/mpirun
```

Comment:
Isn't that automatic?

Reply:
Suprisingly it didn't work. On my other machines the CMake MPI module automatically set this variable but it was not being defined in the travis environment. Adding it fixed the problem.

Reply:
Hmm, the CMake on travis is very new, so maybe the variable got renamed!

Reply:
That would seem odd as I got my instructions from the latest docs: https://cmake.org/cmake/help/latest/module/FindMPI.html

Reply:
I will have a look!

Reply:
Got it, Travis run CMake-3.9 and it is called `MPIEXEC` (not `MPIEXEC_EXECUTABLE`) in that version.

Reply:
`MPIEXEC` is listed under [Backward-Compatibility](https://cmake.org/cmake/help/v3.11/module/FindMPI.html#backward-compatibility), so we should use that one.